community-operators/federation/federation.v0.0.5.clusterserviceversion.yaml

```diff
@@ -0,0 +1,189 @@
+#! validate-crd: deploy/chart/templates/0000_30_02-clusterserviceversion.crd.yaml
+#! parse-kind: ClusterServiceVersion
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  name: federation.v0.0.5
+  namespace: placeholder
+  annotations:
```

Comment:
Are there any example CRs? These can be added to `metadata.annotations.alm-examples` as a json list of objects.

Reply:
Not at this point, and I do not have time to come up with them now.

Reply:
Referring to an earlier conversation we had, there are some example CRs here: https://github.com/openshift/federation-v2-operator/tree/master/vendor/github.com/kubernetes-sigs/federation-v2/example/sample1
I can add these in a separate PR if that helps.